packages/core/src/Transition/FoldTransition.tsx

```diff
@@ -35,6 +35,7 @@ export const FoldTransition: FC<FoldTransitionProps> = ({
     const node = nodeRef.current
 
     const height = node.offsetHeight
+    if (height === 0) return
```

Comment:
It seems like the problem is the `increaseHeight` function is called twise when it opens. Do you know why? If it is called once, I think you don't need to add this line.

Reply:
I am not sure exactly, but when I take away react strict mode, it stops running twice.

Reply:
Ok! I can approve this when you change the commit message.